app/v1/models/request/AddressModel.scala

```diff
@@ -56,24 +58,35 @@ object AddressModel {
     }
 
   }
+  private[models] def validateDateAsPriorDate(earlierDate: DateModel, optionalLaterDate: Option[DateModel]) = {
+    optionalLaterDate.fold(true){ laterDate =>
+      val earlierModelAsDate = LocalDate.parse(earlierDate.dateString, DateTimeFormatter.ofPattern("dd-MM-yyyy"))
+      val laterModelAsDate = LocalDate.parse(laterDate.dateString, DateTimeFormatter.ofPattern("dd-MM-yyyy"))
+      Logger.warn("[NameModel][validateDateAsPriorDate] The provided earlierDate is after the laterDate.")
```

Comment:
This logger message will be thrown every time. It only needs to be thrown when the earlierDate is after the laterDate